whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
-90 deg, the optimal strategy for shear systematics suppression will be to aim for
-uniformity if the square of the cosine of the camera rotation angle.  i.e. +90 deg 
-and -90 deg are degenerate, so we want half as much coverage at those angles than at 0 deg.
-
-Similarly, the telescope optics may harbor systematic aberrations, and
-these also could be mitigated by recording images with varying
-parallactic angle.  Another relevant parameter, RotTelPos, is
-indicative of the projected angle of the telescope optics on the sky. More 
-important is the effect of atmospheric chromatic diffraction: re-visits to a given
-field should be distributed over RA, consistent with airmass and seeing limits.
-
-Uniformity of depth is important, but less so than uniformity in camera
-rotator shear suppression.  Simulations have shown that for the Gold sample of galaxies,
-uniformity at the 0.2 mag level in limiting magnitude produces little shear bias. The
-largest effect comes from bias in weak lens magnification tomography.
+systematics residuals.  Simulations which add camera angle dithering show that
+the goal can be achieved in fields with relatively uniform seeing history
+\citep{Jee&Tyson2011}.
+
+Thus shear systematics will be reduced by randomization of the orientation of
+the camera with respect to the sky.  This is represented by the parameter
+RotSkyPos, defined as the angle between the $+y$ camera direction and North.  We
+can construct diagnostic metrics that quantify the uniformity of its
+distribution at each sky position.  Given the spin 2 symmetry of shear, the
+optimal strategy for shear systematics will be to aim for uniformity of
+RotSkyPos mod $\pi$, since angles separated by $\pi$ radians are degenerate.
+
+Similarly, the telescope optics may harbor systematic aberrations, and these
+also could be mitigated by recording images with a uniform distribution of
+parallactic angle, which is the angle between North and zenith for a given field
+observation.  Differential chromatic refraction (DCR) effects will also be
+mitigated by varying the parallactic angle, though the exact relationship is
+complex since the parallactic angle sets both the DCR magnitude and direction
+for a given observation.  Re-visits to a given field should be distributed over
+parallactic angles (or equivalently, hour angles), consistent with airmass and
+seeing limits.  Note that wide hour angle coverage for a given field will also
+be helpful in order to efficiently achieve full 180 deg coverage in CCD sky
+angle.
+
+As argued below, survey depth is more important than survey area early in the
+survey.  Uniformity of depth is important, but less so than uniformity in camera
+rotator shear suppression.  Simulations have shown that for the Gold sample of
+galaxies, uniformity at the 0.2 mag level in limiting magnitude produces little
+shear bias.  The largest effect comes from bias in weak lensing magnification
+tomography \citet{Morrison2012}.
```

Comment:
I don't quite understand the intent of the last statement.  Is this a distinct science case - i.e., uniformity at 0.2 mag level produces little shear bias for a cosmic shear measurement, while WL magnification tomography requires even better uniformity?